runtime/j9vm31/CMakeLists.txt

```diff
@@ -58,14 +58,28 @@ set(compile_flags_to_remove
 	"\"-Wa,SYSPARM(BIT64)\""
 	"-DJ9ZOS39064"
 	"-qxplink=noback"
+	"-m64"
 )
 
+# Flags to be appended while compiling
+set(compile_flags_to_append
+	"-m32n"
```

Comment:
These options are Open XL specific. Can we add a toolchain check here?

Reply:
@keithc-ca being just a cmake txt file and within the OpenJ9 repo, can I use ifeq( for variables like "OMR_TOOLCONFIG" or "CMAKE_C_COMPILER_IS_OPENXL" which are defined in the OMR repo?
I wasnt too certain about that, and if I were to use the latter, then this PR can only be merged once the initial toolchain PR is merged.

Reply:
It is entirely reasonable to use `OMR_TOOLCONFIG STREQUAL "openxl"` or `OMR_TOOLCONFIG STREQUAL "xlc"` here (even before OMR would ever set `OMR_TOOLCONFIG` to `"openxl"`). On the other hand, we shoud not use `CMAKE_C_COMPILER_IS_OPENXL` until OMR gives meaning to that name.

Reply:
Tested with conditional and pushed changes.